src/environment.cpp

```diff
@@ -202,21 +202,25 @@ void Environment::continueRaycast(RaycastState *state, PointedThing *result)
 			// ID of the current box (loop counter)
 			u16 id = 0;
 
+			/*
+			"I understand how the engines work now.
+			It came to me in a dream. The engines don't move the ship at all.
+			The ship stays where it is and the engines move the universe around it."
+			- Hubert J. Farnsworth in "Futurama"
+			*/
```

Comment:
perhaps this is better put into the new `boxLineCollision()` and just a comment like "note that the calculcations here are done relatively" here?